lib/components/source/SourcePanel.vue

```diff
@@ -16,7 +16,7 @@ v-flex#source-editor(sm4 md6)
 
       refresh-source-button(:spreadsheet="componentSheet")
 
-      v-select(box label="Worksheet" v-model="worksheetId" :items="worksheetOptions" item-value="id" item-text="title")
+      v-select(box label="Worksheet" v-model="worksheetId" :items="worksheetOptions" item-value="id" item-text="title" :error="worksheetId ? false : true")
```

Comment:
Beautiful!

Does this work? `:error="!worksheetId"`

Reply:
That does work, I'll go ahead and make the changes!